index.d.ts

```diff
@@ -1061,6 +1080,13 @@ declare module '@xmldom/xmldom' {
 
 	interface ProcessingInstruction extends Node {
 		nodeType: typeof Node.PROCESSING_INSTRUCTION_NODE;
+		/**
+		 * Everything that goes after the target, excluding `?>`.
```

Comment:
I would probably just use the definition provided by MDN: A string representing the textual data contained in this object.